README.md

```diff
@@ -29,7 +29,17 @@ TL;DR build taco using CMake. Run `make test`.
 # Build and test
 ![Build and Test](https://github.com/RSenApps/taco/workflows/Build%20and%20Test/badge.svg?branch=master)
 
-Build taco using CMake 3.4.0 or greater:
+## Dependencies
+- CMake 2.8.12 or greater
+- GNU Make 4.2.1 or greater
+- conda 4.8.2 or greater
```

Comment:
Given we use conda to download the dependencies, you don't need to specify them here. Maybe just specify conda and the LLVM version we're using (LLVM 11)

Reply:
Done